documentation/src/main/asciidoc/reference/mapper-orm-indexing-jsr352.asciidoc

```diff
@@ -87,6 +87,9 @@ place after indexing.
 The default is `IGNORE`, and it will be the most efficient choice in most cases,
 but using another mode such as `GET` may be more efficient if many of the entities being indexed
 refer to a small set of other entities.
+Enabling caches has an effect only if the entity has some lazy fields and/or the entity id is the document id too.
```

Comment:
I don't think lazy fields have anything to do with this?

```suggestion
Enabling caches has an effect only if the entity id is also the document id, which is the default.
```

Reply:
Actually, I think that there could be some benefit, in the case the entity id was not the document id, but the entity had some lazy fields. But let's not confuse the user with that ;)